tests/specs/multiremote.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('wdio-image-comparison-service check that multi remote is working', () => {
+
+
+    beforeEach(() => {
+        chromeBrowserOne.url('');
+        chromeBrowserOne.pause(500);
+
+        chromeBrowserTwo.url('');
+        chromeBrowserTwo.pause(500);
+    });
+
+    // Chrome remembers the last postion when the url is loaded again, this will reset it.
+    afterEach(() => {
+        chromeBrowserOne.execute('window.scrollTo(0, 0);', [])
+        chromeBrowserTwo.execute('window.scrollTo(0, 0);', [])
+    });
+
+    it('take a screenshot of each browser', () => {
+        //Not a great test, but saves the screen on one browser and expect it to be exactly the same on the other
+        //Note that autoSaveBaseline is set to false but neither of these capabilities have a logname set so they'll use
+        // the same file name given the same tag.
+        chromeBrowserOne.saveScreen('homepage')
```

Comment:
Thanks for the tests 🎉 

When you fixed the `capabilities` issue as mentioned above can you then create a test like this

```js
chromeBrowserOne.saveScreen('homepage')
chromeBrowserTwo.saveScreen('homepage')
```

and then a test like this
https://github.com/wswebcreation/wdio-image-comparison-service/blob/master/tests/specs/basics.spec.js#L25
where you verify that the file has been written with the correct name?

Reply:
Alright I will add that then to make sure things look good.